src/main/java/com/starcallingassist/sidepanel/TableHeader.java

```diff
@@ -51,17 +50,16 @@ class TableHeader extends JPanel
     private static final Color ARROW_COLOR = ColorScheme.LIGHT_GRAY_COLOR;
     private static final Color HIGHLIGHT_COLOR = ColorScheme.BRAND_ORANGE;
 
-    static
-    {
-	final BufferedImage arrowDown = ImageUtil.loadImageResource(StarCallingAssistPlugin.class, "/arrow_down.png");
-	final BufferedImage arrowUp = ImageUtil.rotateImage(arrowDown, Math.PI);
-	final BufferedImage arrowUpFaded = ImageUtil.luminanceOffset(arrowUp, -80);
-	ARROW_UP = new ImageIcon(arrowUpFaded);
-
-	final BufferedImage highlightArrowDown = ImageUtil.fillImage(arrowDown, HIGHLIGHT_COLOR);
-	final BufferedImage highlightArrowUp = ImageUtil.fillImage(arrowUp, HIGHLIGHT_COLOR);
-	HIGHLIGHT_ARROW_DOWN = new ImageIcon(highlightArrowDown);
-	HIGHLIGHT_ARROW_UP = new ImageIcon(highlightArrowUp);
+    static {
```

Comment:
Keep static on a newline
```suggestion
    static
    {
```